.circleci/config.yml

```diff
@@ -51,5 +52,10 @@ workflows:
           vcs-type: << pipeline.project.type >>
           pipeline-number: << pipeline.number >>
           filters:
+            branches:
```

Comment:
lets remove the filter then remove the test-deploy jobs and commands being tested